bankruptcy_model/data_visualizations/pca.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
```

Comment:
Jak wykorzystujesz nowe biblioteki to dodaj je do requirements.txt żeby można je było łatwo doinstalować